docs/UserWallet.rst

```diff
@@ -83,16 +89,32 @@ Each wallet uses a pair of public and private keys. The public key is used ad th
 .. Warning:: Remember that losing your private keys means losing all your crypto-money. You must take precautions and be very careful! Even an administrator won't be able to help you if it gets compromised or lost.
 
 
+Wallet password
+----------------
+
+You can change your wallet password by clicking on "Change password" button from security popup. You just need to enter your current password and the new password then confirm.
+
+|image20|
+
+The option "Remember my password on this browser" stores your password in the browser’s local storage to be automatically used for every transaction (including purchases in the Perk Store) without requiring your input each time. 
+
+|image21|
+
+You must enter your current password to activate this option.
+
+|image22|
+
+
 How to send funds from wallet
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 To send funds from your wallet, you have to click on "Send" button then, in the "send funds" popup, select a recipient, your wallet password and the amount to send. You can optionally type a label and a message that will be stores in the transaction history.
 You also have to choose the type of transaction between three options: "Cheap", "Normal" or "Fast". 
-The difference between those types of transaction is the price and the rapidity of transaction:
+The difference between those types of transaction is the price and the speed of processing transaction in the blockchain network. There is no guaranteed delays for transaction processing because it depends on how much the blockchain network is busy, but basically, the more you pay, the faster. Below are some indicative delays to expect :
```

Comment:
```suggestion
The difference between these transaction modes is the fees and the speed of processing transaction in the blockchain network. There is no guaranteed delay for transaction processing because it depends on how much the blockchain network is busy. But basically, the more you pay, the faster. Below are some indicative delays to expect :
```